### ISSUE TITLE
Multiple pawns on the same single ICE protecting a server don't work

### ISSUE THREAD

**Issue by zaroth** [mtgred/netrunner#1021]
In general, Exile Pawn Deep Red archetype needs some love and implementation. Currently the level of automation doesn't make the deck very playable. Example deck: http://netrunnerdb.com/en/decklist/29269/street-tech-chess

**Comment by zaroth** [comment]
Also, there is a considerable slowdown/freeze when making succesful runs with multiple pawns installed - possible resource-intensive implementation?

**Comment by pjbrand** [comment]
Seconding this issue, but furthermore for me even single pawns weren't progressing into the central (and being trashed) as they should be. Not sure if it's working from ice-to-ice since the server I had slapped them on only had 1 ice. I did, however, later put two Pawn onto that same ice and they both did not trigger correctly upon a successful run. 

Would appreciate some love for this, even though Exile and Pawn are niche cards. Luckily I was on Skype with my opponent so he understood why the manual trashing/dragging/drawing was going on; Deep Red Exile is so much fun.
